evap/staff/views.py

```diff
@@ -356,51 +347,46 @@ def course_review(request, semester_id, course_id, offset=None):
         messages.warning(request, _("Reviewing not possible in current state."))
         return redirect('evap.staff.views.semester_view', semester_id)
 
-    reviewFS = modelformset_factory(TextAnswer, form=ReviewTextAnswerForm, can_order=False, can_delete=False, extra=0)
-
-    # compute base queryset
-    base_queryset = course.textanswer_set.filter(checked=False).values_list('id', flat=True).order_by('id')
+    review_formset = modelformset_factory(TextAnswer, form=ReviewTextAnswerForm, can_order=False, can_delete=False, extra=0)
 
-    # figure out offset
-    if offset is None:
-        # get offset for current course
-        key_name = "course_%d_offset" % course.id
-        offset = cache.get(key_name) or 0
-
-        # store offset for next page view
-        cache.set(key_name, (offset + TextAnswer.elements_per_page) % base_queryset.count())
-    else:
-        offset = int(offset)
+    skipped_answers = request.POST.get("skipped_answers", "") if request.POST else request.GET.get("skipped_answers", "")
```

Comment:
The skipped parameter is read from the URL here - is there a way to make this more explicit?
It is needed because we're doing a redirect to the same view when opening the next review-form